src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java

```diff
@@ -34,7 +34,7 @@
 	public void visitPatientDashboard() {
 		testPatient = createTestPatient();
 		initiateWithLogin();
-		findPatientPage = (FindPatientPage) homePage.goToFindPatientRecord().waitForPage();
+		findPatientPage = (FindPatientPage) homePage.goToFindPatientRecord();
```

Comment:
@Irenyak1 Hello  think you need to take care of these pages, Like how @kdaud  suggested, there are certain pages you will need to wait to finish up the functionalities.